src/components/blocks/DetailedViewStagingTab.js

```diff
@@ -23,48 +21,48 @@ const StyledDetailedViewTab = styled('div')`
   gap: 20px;
 `;
 
-const StyledWordWrap = styled('div')`
+export const StyledWordWrap = styled('div')`
   word-break: break-word;
   display: inline-block;
 `;
 
 const DetailedViewTabItem = ({ entry, action }) => {
-  const hasEntryKeyData = key =>
-    (entry[key].original !== null && entry[key].original !== 'null') ||
-    (entry[key].changes &&
-      entry[key].changes.some(change => change !== null && change !== 'null'));
-
-  const isKeyAllowedToBeDisplayed = key =>
-    ![
-      'orgUid',
-      'warehouseProjectId',
-      'id',
-      'createdAt',
-      'updatedAt',
-      'label_unit',
-    ].includes(key);
+  // const hasEntryKeyData = key =>
```

Comment:
by the time this PR is ready to merge makes sure all the commented out code is gone